docs/_docs/interface-localization.md

```diff
@@ -1,63 +0,0 @@
----
-layout: article
-title: Interface Localization
-category: admin
----
```

Comment:
Seems like this file vanished from the new documentation.

Reply:
Yes, I've deleted the file

Reply:
In the future, please highlight such removal and describe why you think it should happen.